tests/core/test_open_in_process.py

```diff
@@ -211,6 +211,28 @@ async def _do_inner():
     await asyncio.wait_for(_do_inner(), timeout=1)
 
 
+@pytest.mark.asyncio
+async def test_open_proc_unpickleable_exc(open_in_proc):
+    # Custom exception classes requiring multiple arguments cannot be pickled:
+    # https://bugs.python.org/issue32696
+    # Need to find a way to workaround that so that this test passes
+    class CustomException(BaseException):
+        def __init__(self, msg, arg2):
+            super().__init__(msg)
```

Comment:
Would not calling `super().__Init(mst)` (and instead setting the `msg` locally)  help?

Reply:
It doesn't, but even if it did make the test pass, any custom exceptions with extra required `__init__()` args would cause us to crash.

Reply:
Are we hitting this somewhere in Trinity?

Reply:
Yes, any uncaught `rlp.exceptions.SerializationError`s: https://github.com/ethereum/trinity/issues/2023